CHANGELOG.md

```diff
@@ -32,7 +31,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - SFT/SteerLM: LoRA can now be enabled on all model layers
 - DPO: Enable LoRA on all model layers (In this case the actor will be reference model + LoRA weights, we can switch between actor/reference model by enabling/disabling LoRA)
 - PPO: Enable LoRA on all model layers (In this case the actor will be init policy + LoRA weights, we can switch between actor/init_policy model by enabling/disabling LoRA)
-- SteerLM 2.0: Add the SteerLM 2.0 model alignment method.
 - Added support for float values for `val_check_interval` for SFT
```

Comment:
suggested revision to remove multiple "for".

Added `val_check_interval` float values for SFT.

Reply:
I think the new phrasing sounds like a different feature. What about:
```suggestion
- `val_check_interval` in SFT now supports float values
```
?

Reply:
Fix as you need. This was just a suggestion.